docker/elasticsearch6/elasticsearch.repo

```diff
@@ -0,0 +1,6 @@
+[elasticsearch-kibana-logstash-6.x]
```

Comment:
Need to avoid this for Monasca - use ELK 5.

Reply:
thanks

Reply:
This is a separate image. The elasticsearch image still uses 5.x.